package.json

```
@@ -14,11 +14,13 @@
     "build:frontend": "yarn workspace @asap-hub/frontend build",
     "build:frontend:typecheck": "yarn workspace @asap-hub/frontend typecheck",
     "build": "yarn build:typecheck && yarn build:babel && yarn build:frontend:typecheck && yarn build:frontend",
+    "watch:babel": "yarn workspaces foreach -pi -j 1000 run watch:babel",
+    "watch:typecheck": "yarn build:typecheck -w",
+    "dev": "yarn sls offline",
```

```suggestion
    "start:backend": "yarn sls offline",
```
my suggestion - `start` would be consistent with CRA, and backend clarifies which part it starts